vector/src/main/res/values/strings.xml

```diff
@@ -3388,4 +3391,5 @@
     <string name="this_space_has_no_rooms_not_admin">Some rooms may be hidden because they’re private and you need an invite.\nYou don’t have permission to add rooms.</string>
     <string name="this_space_has_no_rooms_admin">Some rooms may be hidden because they’re private and you need an invite.</string>
 
+    <string name="unnamed_room">Unnamed Room</string>
```

Comment:
The wording is OK (I think), but I'm wondering if in French I would translate it to "Room anonyme" which is a bit strange to me, so I would go for "Room sans nom", so I'm wondering if "Room without name" would be better here...

Reply:
It was to be consistent with web, not sure what would be better.
Anyhow, now web is checking if that room is known locally to get the good name so this would happen less often